common/templates/common/_category_methodology_section.html

```diff
@@ -2,7 +2,7 @@
 
 <h2 class="methodology-section-header">Data Tracked For This Category</h2>
 
-<div class="methodology-summary">{{ methodology.description }}</div>
+<div class="methodology-summary rich-text">{{ methodology.description|richtext }}</div>
```

Comment:
I know this isn't a thing you added, but now I'm really confused by what this `.description` is doing here. `methodology` is a rich text field, right? What does adding `.description` do? Can we remove it? If I remove it locally, the output doesn't seem to change.

Reply:
Good point. I think in the original PR, some unecessary bit got left behind. removed it.